### PR TITLE
[plugin-web-app] Update dropdown steps syntax

### PR DESCRIPTION
**Pull request by PSharuba**

### CODE REVIEW

**Review by valfirst — CHANGES_REQUESTED** [review]

File: vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/DropdownSteps.java [hunk 1]
```diff
@@ -146,21 +252,60 @@ public void addOptionInDropdown(String option, Locator locator)
...
     private void runIfDropdownExists(Locator locator, Consumer<Select> toRun)
     {
         findDropdown(locator).ifPresent(toRun);
     }
 
     private Optional<Select> findDropdown(Locator locator)
     {
-        return Optional.ofNullable(baseValidations.assertIfElementExists("Dropdown", locator))
+        return baseValidations.assertElementExists(DROPDOWN, locator)
                        .map(Select::new);
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/DropdownSteps.java:309]:
```suggestion
        return baseValidations.assertElementExists(DROPDOWN, locator).map(Select::new);
```

File: vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/DropdownSteps.java [hunk 2]
```diff
@@ -146,21 +252,60 @@ public void addOptionInDropdown(String option, Locator locator)
...
 
+    private void runIfDropdownExistsDeprecated(Locator locator, Consumer<Select> toRun)
+    {
+        findDropdownDeprecated(locator).ifPresent(toRun);
+    }
+
+    private Optional<Select> findDropdownDeprecated(Locator locator)
+    {
+        return Optional.ofNullable(baseValidations.assertIfElementExists(DROPDOWN, locator))
+                       .map(Select::new);
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/DropdownSteps.java:293]:
```suggestion
        return Optional.ofNullable(baseValidations.assertIfElementExists(DROPDOWN, locator)).map(Select::new);
```

**Review by valfirst — CHANGES_REQUESTED** [review]

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 3]
```diff
@@ -468,6 +468,12 @@ When I clear field located by `id(password)` using keyboard
 Validates whether the dropdown located by locator exists and contains the list
 of the expected options.
 
+[source,gherkin]
+----
+Then dropdown located by `$locator` contains options: $option
```
Comment (uarlouski) [inline docs/modules/plugins/pages/plugin-web-app.adoc:473]: lets remove the space before $options since it doesn't bring any value

Comment (valfirst) [inline docs/modules/plugins/pages/plugin-web-app.adoc:473]:
```suggestion
Then dropdown located by `$locator` contains options:$options
```

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 4]
```diff
@@ -468,6 +468,12 @@ When I clear field located by `id(password)` using keyboard
...
+----
+
+_Deprecated syntax (will be removed in VIVIDUS 0.7.0)_:
 [source,gherkin]
 ----
 Then dropdown located `$locator` contains options: $option
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-web-app.adoc:479]:
```suggestion
Then dropdown located `$locator` contains options:$options
```

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 5]
```diff
@@ -470,15 +470,21 @@ of the expected options.
 
 [source,gherkin]
 ----
-Then dropdown located `$locator` contains options: $option
+Then dropdown located by `$locator` contains options:$option
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-web-app.adoc:473]:
```suggestion
Then dropdown located by `$locator` contains options:$options
```

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 6]
```diff
@@ -470,15 +470,21 @@ of the expected options.
...
+----
+
+_Deprecated syntax (will be removed in VIVIDUS 0.7.0)_:
+[source,gherkin]
+----
+Then dropdown located `$locator` contains options:$option
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-web-app.adoc:479]:
```suggestion
Then dropdown located `$locator` contains options:$options
```

**Review by valfirst — CHANGES_REQUESTED** [review]

**Review by valfirst — APPROVED** [review]

**Review by uarlouski — APPROVED** [review]

**Pull request merged**